packages/vscode-extension/src/commandsMap.ts

```diff
@@ -3,64 +3,92 @@
  ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
  */
 
-export const commandsInfoMap: {
-	[commandName: string]: {
-		cliCommandName: string;
-		vscodeCommandName: string;
-		// not really needed, just left here as an idea
-		vscodeShortName?: string;
-	};
-} = {
+export type CommandInfo = {
+	vscodeContributedCommand: string;
```

Comment:
Could be just rename it to something like 
vscodeCommandId?

Or we could call this vscodeCommandName and the other one call it vscodeCommandLabel

Reply:
Renamed to vscodeCommandId.